frontend/src/store/shoot/helper.js

```diff
@@ -414,6 +414,8 @@ export function getRawVal (context, item, column) {
       return getIssueSince(item.status) || 0
     case 'technicalId':
       return item.status?.technicalID
+    case 'workers':
+      return item.spec.provider.workers?.length || 'workerless'
```

Comment:
why this magic value? so that you can search for it?

Reply:
removed, added explicit code in search function to filter for workerless shoots